source/game.c

```diff
@@ -3153,10 +3272,21 @@ static void jokers_sel_row_on_selection_changed(SelectionGrid *selection_grid,
         JokerObject* joker_object = (JokerObject*)list_get_at_idx(&_owned_jokers_list, new_selection->x);
         if(joker_object != NULL)
         {
-            sprite_object_set_focus(joker_object->sprite_object, true);
-            print_price_under_sprite_object(joker_object->sprite_object, joker_get_sell_value(joker_object->joker));
+            if (!swapping)
+            {
+                sprite_object_set_focus(joker_object->sprite_object, true);
+            }
+            if (!key_is_down(SELECT_CARD))
+            {
+                print_price_under_sprite_object(joker_object->sprite_object, joker_get_sell_value(joker_object->joker));
+            }
```

Comment:
```suggestion
            if (!key_is_down(SELECT_CARD))
            {
                print_price_under_sprite_object(joker_object->sprite_object, joker_get_sell_value(joker_object->joker));
            }
```

Why not put it together wit the above swapping condition? It will mean the price will be displayed for the case of entering the joker row while the key is held will display the price despite being in swapping mode but I guess it's not so bad since you technically can sell the joker while swapping and the call to `erase_price_under_sprite_object();` executed even when swapping in the `prev_selection` condition means the price will be erased when swapping. Maybe it's good to add a comment there explaining it's needed for this case... I'm starting to think my request to erase the price while swapping made the code a little messy...

Reply:
I think adding a comment should be fine, it's a bit convoluted but I don't think it's too messy yet